packages/sdk/src/bot/teamsBotSsoPrompt.ts

```diff
@@ -348,7 +350,7 @@ export class TeamsBotSsoPrompt extends Dialog {
       this.settings.scopes.join(" ")
     )}&clientId=${config.authentication!.clientId}&tenantId=${
       config.authentication!.tenantId
-    }&email=${email}`;
+    }&loginHint=${loginHint}`;
```

Comment:
what will happen if some implementation does not pass loginHint? By the way do we need to make loginHint optional? Since it's a private function.

Reply:
If the loginHint is empty, the sign in page will let user choose account to sign in. So makes loginHint optional in case btobuilder TeamsInfo SDK fails to get member's userPrincipalName.